src/views/CoCurricularTranscript/index.tsx

```diff
@@ -20,30 +21,31 @@ const SectionTitle: { [Key in keyof TranscriptItems]: string } = {
 const CoCurricularTranscript = () => {
   const [loading, setLoading] = useState(true);
   const [transcriptItems, setTranscriptItems] = useState<TranscriptItems | undefined>();
-  const { profile, loading: loadingProfile } = useUser();
+  const isAuthenticated = useIsAuthenticated();
+  const { profile: userProfile, loading: loadingProfile } = useUser();
+  const profile = isAuthenticated ? userProfile : exampleStudentProfile;
 
   useEffect(() => {
     const loadTranscript = async () => {
-      if (!profile) return;
-
-      setLoading(true);
-
-      await transcriptService.getItems(profile.AD_Username).then(setTranscriptItems);
-
+      if (isAuthenticated) {
+        if (!profile) {
+          return;
+        }
+        const transcriptItems = await transcriptService.getItems(profile.AD_Username);
+        setTranscriptItems(transcriptItems);
+        setLoading(true);
```

Comment:
Having `setLoading(true)` here is confusing. We want the page to be loading while we're waiting on data, right? So shouldn't we set loading to true before we start loading data, i.e. above the `await` line?